src/Controller/InspectController.php

```diff
@@ -73,11 +73,19 @@ public function getTranslations(ContainerInterface $container): ResponseInterfac
         }
         $messages = [];
         foreach ($categorySources as $categorySource) {
-            $messages[$categorySource->getName()] = [];
+            if (!isset($messages[$categorySource->getName()])) {
```

Comment:
```suggestion
            $categoryName = $categorySource->getName();
            if (!isset($messages[$categoryName])) {
```

Please reuse the category name variable down to code